Sources/CoreMetrics/SystemMetrics.swift

```diff
@@ -0,0 +1,172 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Metrics API open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the Swift Metrics API project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Metrics API project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Combine
```

Comment:
Since when could we import this on Linux?

Reply:
We can't (-:

Sorry @MrLotU but we'll have to do without Combine. (even if we could, I don't think this library should pull it in)